providers/efa/man/efadv_query_ah.3.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+title: EFADV_QUERY_AH
+section: 3
+tagline: Verbs
+date: 2019-05-19
+header: "EFA Direct Verbs Manual"
+footer: efa
+---
+
+# NAME
+
+efadv_query_ah - Query EFA specific Address Handle attributes
+
+# SYNOPSIS
+
+```c
+#include <infiniband/efadv.h>
+
+int efadv_query_ah(struct ibv_ah *ibvah, struct efadv_ah_attr *attr);
+```
+
+# DESCRIPTION
+
+**efadv_query_ah()** queries device-specific Address Handle attributes.
+
+Compatibility is handled using the comp_mask field which serves as both an
+indication for requested attributes and returned attributes.
+
+```c
+struct efadv_ah_attr {
+	uint8_t version;
+	uint8_t reserved[7];
+	uint64_t flags;
+	uint64_t comp_mask;
+	uint16_t ahn;
+};
+```
+
+*comp_mask*
+:	Compatibility mask.
+
+# RETURN VALUE
+
+**efadv_query_ah()** returns 0 on success, or the value of errno on failure
+(which indicates the failure reason).
+
+# SEE ALSO
+
+**efadv**(7)
+
+# NOTES
+
+* Compatibility mask (comp_mask) is an in/out field.
+
```

Comment:
Same remarks as the other man page

Reply:
Done

Reply:
More clarity. comp_mask is an out and currently has no values

It is more conventional to pass the size of the structure as a function argument not embedded in the structure itself.